src/PushNotifications.php

```diff
@@ -13,6 +14,9 @@ class PushNotifications {
   const MAX_INTERESTS = 100;
   const MAX_INTEREST_LENGTH = 164;
   const INTEREST_REGEX = "/^(_|-|=|@|,|\\.|;|[A-Z]|[a-z]|[0-9])+$/";
+  const MAX_USERS = 1000;
+  const MAX_USER_ID_LENGTH = 164;
+  const AUTH_TOKEN_DURATION = 24 * 60 * 60;
```

Comment:
Adding the unit to any variable holding a time/duration is usually helpful, e.g. `AUTH_TOKEN_DURATION_SECS`